### PR TITLE
Update stale documentation about IndexedDB module caching

### PR DESCRIPTION
**Pull request by jnickg**
According to some reading on the web, this documentation was out of date. Just giving a minor change to better capture the current state of affairs.

### CODE REVIEW

File: site/source/docs/compiling/Deploying-Pages.rst [hunk 1]
```diff
@@ -56,7 +56,7 @@ While the first run experience of visiting a page can take some time to finish a
...
 
-- Compilation results of asm.js modules are cached automatically by the browser, and there is little control over this. WebAssembly on the other hand supports explicit caching of compiled ``WebAssembly.Module`` objects to IndexedDB. This feature should be always leveraged, since it allows skipping the whole compilation process on the second page visit.
+- Compilation results of asm.js modules are cached automatically by the browser, and there is little control over this. WebAssembly does not yet have automatic caching support among all modern browsers, though Chromium-based browsers do (read more `here <https://v8.dev/blog/wasm-code-caching>`_). Previously, manual caching of compiled modules via IndexedDB was recommended; however, this is now largely unsupported (read more `here <https://github.com/WebAssembly/spec/issues/821>`_).
```
Comment (sbc100) [inline site/source/docs/compiling/Deploying-Pages.rst:59]: Can you also remove the first sentence here.. we don't support asm.js anymore.

Comment (sbc100) [inline site/source/docs/compiling/Deploying-Pages.rst:59]: Can you clarify that you are talking about compiled webassembly modules.  The `.wasm` files downloaded from the network *are* cached by the browser like any other resource right?

**Review by sbc100 — COMMENTED** [review]
Thanks!

File: site/source/docs/compiling/Deploying-Pages.rst (hunk 1 above)
Comment (jnickg) [reply site/source/docs/compiling/Deploying-Pages.rst:59]: Yes, that is a good point. Will do 👍

Comment (jnickg) [reply site/source/docs/compiling/Deploying-Pages.rst]: Done.

**Review by sbc100 — APPROVED** [review]

**Pull request merged**